app/templates/index.html

```diff
@@ -2,7 +2,17 @@
 {% block title %}Home{% endblock %}
 {% block body %}
 <div class="jumbotron">
-  <h1>Flask Is Awesome</h1>
-  <p class="lead">And I'm glad to be learning so much about it!</p>
+    <h1>What is Genten Studios?</h1>
+    <p class="lead">Genten Studios is a group of programmers who are passionate about creating games. All of our games
+        follow a common pattern that the program itself does not provide any content. Content is defined by modules
+        written in Lua which the game does the heavy lifting turning it into something playable. In the long-term our
+        goal is to create Quartz Engine to be the core of the games we create in this style.</p>
+    <h2>Project Phoenix</h2>
+    <p>Our project coded "Phoenix" is the first game that we are developing in this style. Phoenix is an open world
+        sandbox game where the player can manipulate anything in a cubic world around them. Check out our source code
+    <a href="https://github.com/GentenStudios/Phoenix">here</a>.</p>
+    <h2>Quartz Engine</h2>
+    <p>Once Project Phoenix has hit an alpha state, the development of Quartz Engine is planned to begin. Quartz Engine
+        will serve as a common library shared between Phoenix and future games we develop in this same style.</p>
```

Comment:
"style" Design principle (Check above comments for context)

Reply:
:heavy_check_mark: